README.md

```diff
@@ -89,7 +89,7 @@ automatically on reboots.
 Install the following software dependencies:
 
     $ sudo apt-get update
-    $ sudo apt-get -y -q install git clang devscripts doxygen hugepages \
+    $ sudo apt-get -y -q install git clang devscripts doxygen libhugetlbfs-bin \
```

Comment:
Add a comment in the text below that until Ubuntu 18.04, the name of the package `libhugetlbfs-bin` is `hugepages`. This information is going to be useful to some.